{
  try {
    doHandle(exchange);
  }
 catch (  Exception e) {
    sendResponse(exchange,INTERNAL_SERVER_ERROR,e.getMessage() + "\n");
  }
}
